tasks/freeradius-mysql.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Freeradius additional mysql system packages
+  when: openwisp2_radius and freeradius_sql.dialect == "mysql"
+  apt:
+    name:
+      - freeradius-mysql
+      - mariadb-server
+    state: latest
+  notify: start mysql
+
+- name: Install pymysql
+  when: openwisp2_radius and freeradius_sql.dialect == "mysql"
+  pip:
+    name: pymysql
```

Comment:
what is `pymysql` for? Is it for django or for freeradius? I don't think freeradius uses pymysql.

Reply:
keep in mind that the python DB drivers are already installed in the `pip.yml` tasks:
https://github.com/openwisp/ansible-openwisp2/blob/694e7198bfa3dbb8ca42d168a46c3a84af4fe21d/tasks/pip.yml#L210-L236

Reply:
Removed file! :-)